<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Final Fantasy X Relationship Tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="yunaImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/yuna.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="auronImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/auron.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="kimariImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/kimari.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="wakkaImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/wakka.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="luluImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/lulu.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rikkuImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/rikku.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="yunaText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="auronText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="kimariText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wakkaText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="luluText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rikkuText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
